src/web/crate_details.rs

```diff
@@ -139,6 +145,16 @@ impl CrateDetails {
                 .collect()
         };
 
+        let github_metadata = if krate.get::<_, Option<String>>("github_repo").is_some() {
```

Comment:
This is a change in behavior, right? Before it would have `github = true` whenever you had `http://github.com` as the URL in Cargo.toml, now it will only have it when the repository exists (not private or deleted).

I think that's a good change, just wanted to double check.

Reply:
Yes.